wp-shopping-cart.php

```diff
@@ -123,11 +123,9 @@ public function setup_table_names() {
 		global $wpdb;
 		$wpdb->wpsc_meta                = WPSC_TABLE_META;
 		$wpdb->wpsc_also_bought         = WPSC_TABLE_ALSO_BOUGHT;
-		$wpdb->wpsc_region_tax          = WPSC_TABLE_REGION_TAX;
 		$wpdb->wpsc_coupon_codes        = WPSC_TABLE_COUPON_CODES;
 		$wpdb->wpsc_cart_contents       = WPSC_TABLE_CART_CONTENTS;
 		$wpdb->wpsc_claimed_stock       = WPSC_TABLE_CLAIMED_STOCK;
-		$wpdb->wpsc_currency_list       = WPSC_TABLE_CURRENCY_LIST;
```

Comment:
Why are these tables removed?  We talked in Slack about back compat concerns here...I might be missing something though?